sklearn/cross_decomposition/tests/test_pls.py

```diff
@@ -72,7 +71,7 @@ def test_sanity_check_pls_regression():
     Y = d.target
 
     pls = PLSRegression(n_components=X.shape[1])
-    pls.fit(X, Y)
+    X_trans, Y_trans = pls.fit_transform(X, Y)
```

Comment:
X_trans, Y_trans are not used. Revert this change ?

Reply:
Yes. FYI, I was using them to create a new test but it failed: https://github.com/scikit-learn/scikit-learn/pull/21585#discussion_r744255708

We should open a follow-up issue linked to this.